README.md

```diff
@@ -31,19 +31,31 @@ This repository contains a `copier` template for battery modeling projects using
 
 ## 📄 Using `pybamm-cookiecutter`
 
-### Generating projects with `pybamm-cookiecutter` manually using copier
+### Generating projects with `pybamm-cookiecutter`
 
-This template is not on PyPI yet, so it cannot be installed through `pip` until the first release. Meanwhile, it can be used by cloning this repository and using `copier` to generate a project with this template.
+#### Manually using copier
 
 Install `copier` and `jinja2_time` extension using `pip`.
 ```bash
 pip install copier jinja2-time
 ```
 Generate a project from the `pybamm-cookiecutter` template.
-**Note:** This requires an internet connection. You could manually clone the git repository and run the copy command, or just execute the copy command with the URL to the git repository.
 
 ```bash
-copier copy https://github.com/pybamm-team/pybamm-cookiecutter.git name_of_your_project/ --trust
+copier copy https://github.com/pybamm-team/pybamm-cookiecutter.git . --trust
+# this will generate the project in the current working directory
+copier copy https://github.com/pybamm-team/pybamm-cookiecutter.git path_to_copy_to/ --trust
+# this will generate the project in the specified path
+```
+#### Using pipx
```

Comment:
```suggestion
#### Using pipx (recommended)
```

Maybe we can add a small sentence below this heading as well and link to https://pipx.pypa.io/stable/